torch/nn/utils/_per_sample_grad.py

```diff
@@ -0,0 +1,40 @@
+from torch.nn.utils._stateless import functional_call
+from torch._expanded_weights.expanded_weights_impl import ExpandedWeight
+
+# dependency on `functional_call` means that this can't be exposed in utils
+# without creating circular dependency
+def per_sample_call(module, batch_size, args, kwargs=None):
```

Comment:
nit: we might want to bikeshed this name some more. I would be really confused if I saw `per_sample_call(model)` in someone's code if I didn't already know about per-sample-gradients

Reply:
Yeah definitely see that--what about `call_with_per_sample_gradients`?